integration_tests/compute_runtime/main_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 Fastly, Inc.
+package main
+
+import (
+  "testing"
+  "time"
+
+  "github.com/fastly/compute-sdk-go/compute"
+)
+
+func TestGetVcpuMs(t *testing.T) {
+  start, err := compute.GetVCPUTime()
+  if err != nil {
+    t.Errorf("Couldn't get starting vcpu time")
+  }
+
+  time.Sleep(5 * time.Second)
+
+  end, err := compute.GetVCPUTime()
+  if err != nil {
+    t.Errorf("Couldn't get ending vcpu time")
+  }
+
+  if end - start > time.Second {
+    t.Errorf("Sleeping shouldn't count as vcpu time!")
+  }
+
+  now, err := compute.GetVCPUTime()
+  if err != nil {
+    t.Errorf("Couldn't get starting vcpu time")
+  }
+
+  var counter uint64
+  var next time.Duration
+
+  counter = 0
+  next = now
```

Comment:
```suggestion
  counter := 0
  next := now
```

Reply:
A plain `0` won't have `uint64` type.  (But I +1 the `next := now` tweak).

Reply:
OK, I think 22cbc93 is the combination of your suggestions?